### PR TITLE
fix(best practice guides): new overview doc

### PR DESCRIPTION
**Pull request by zuluecho9**
NR-84528
Work done: 
Remove best practice guides from 'Guides' nav
Create 'Overview of best practice guides' that links out to the docs

### CODE REVIEW

File: src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/overview-best-practices.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Overview of best practice guides
```
Comment (a-sassman) [inline src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/overview-best-practices.mdx:2]: @zuluecho9 Should it be "Best practices guides"?

File: src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/overview-best-practices.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Overview of best practice guides 
+tags:
+  - New Relic solutions
+  - Best practices guides
+metaDescription: A list of New Relic best practice guides.
```
Comment (a-sassman) [inline src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/overview-best-practices.mdx:6]:
```suggestion
metaDescription: A list of New Relic best practices guides. 
```

File: src/content/docs/new-relic-solutions/overview.mdx [hunk 3]
```diff
@@ -3,23 +3,22 @@ title: Solutions, guides, and best practices
...
-
-Using our proven methodology, recommendations, and expert guidance, your organization can move faster, stay on course, and transform successfully! Get started with our guides: 
+Here's an overview of our guides: 
 
 * **Set up New Relic**: see [our Install doc](/docs/new-relic-solutions/new-relic-one/install-configure/install-new-relic) for options.
-* **Learn about the platform**: want to understand common platform experiences and features? See [our platform UI docs](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). 
-* **Best practices guides**: [Learn best practices for some key New Relic features](/docs/new-relic-solutions/best-practices-guides).
-* **Observability maturity**: [Our observability maturity series](/docs/new-relic-solutions/observability-maturity/introduction) identifies common observability goals and teaches you how to meet them. 
+* **Learn about the platform**: want to understand common platform experiences and features? [Get to know the platform.](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one)
```
Comment (a-sassman) [inline src/content/docs/new-relic-solutions/overview.mdx:19]:
```suggestion
* **Learn about the platform**: want to understand common platform experiences and features? [Get to know the platform](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one).
```

File: src/nav/alerts-applied-intelligence.yml [hunk 4]
```diff
@@ -3,6 +3,8 @@ path: /docs/alerts-applied-intelligence
 pages:
   - title: Introduction to alerts and applied intelligence
     path: /docs/alerts-applied-intelligence/overview
+  - title: Alerting best practice guide
```
Comment (a-sassman) [inline src/nav/alerts-applied-intelligence.yml:6]:
```suggestion
  - title: Alerting best practices guide
```

File: src/nav/apm.yml [hunk 5]
```diff
@@ -3,6 +3,8 @@ path: /docs/apm
 pages:
   - title: Introduction to APM
     path: /docs/apm/new-relic-apm/getting-started/introduction-apm
+  - title: APM best practice guide
```
Comment (a-sassman) [inline src/nav/apm.yml:6]:
```suggestion
  - title: APM best practices guide
```

File: src/nav/browser.yml [hunk 6]
```diff
@@ -35,6 +33,14 @@ pages:
         path: /docs/browser/new-relic-browser/configuration/rename-browser-apps
       - title: Delete browser apps
         path: /docs/browser/new-relic-browser/installation/delete-apps-new-relic-browser
+  - title: Guides
+    pages: 
+      - title: Browser best practice guide
```
Comment (a-sassman) [inline src/nav/browser.yml:38]:
```suggestion
      - title: Browser monitoring best practices guide
```

File: src/nav/apm.yml [hunk 7]
```diff
@@ -210,7 +212,7 @@ pages:
                 path: /docs/apm/agents/java-agent/instrumentation/monitor-deployments-java-agent
               - title: Instrument webpages via API
                 path: /docs/apm/agents/java-agent/instrumentation/instrument-browser-monitoring-java-agent-api
-              - title: Browser best practices
+              - title: Browser monitoring best practices
```
Comment (a-sassman) [inline src/nav/apm.yml:215]:
```suggestion
              - title: Browser monitoring best practices in Java
```
Maybe adding the "in Java" will help clarify the difference between this one and the one in the browser monitoring docs.

File: src/nav/browser.yml (hunk 6 above)
Comment (a-sassman) [inline src/nav/browser.yml:38]: Or this to parallel the others
```suggestion
      - title: Guide: Monitoring browser apps 
```

File: src/nav/new-relic-solutions.yml [hunk 8]
```diff
@@ -91,6 +91,8 @@ pages:
         path: /docs/new-relic-solutions/overview
       - title: New Relic glossary
         path: /docs/new-relic-solutions/get-started/glossary
+      - title: Best practice guides
```
Comment (a-sassman) [inline src/nav/new-relic-solutions.yml:94]:
```suggestion
      - title: Best practices guides
```

**Review by a-sassman — DISMISSED** [review]

File: src/content/docs/new-relic-solutions/overview.mdx (hunk 3 above)
Comment (zuluecho9) [reply src/content/docs/new-relic-solutions/overview.mdx:19]: i could be wrong on this, but when I do a full sentence link I include the period. I don't often do full sentences as links but that's what I do in that case.

Comment (a-sassman) [reply src/content/docs/new-relic-solutions/overview.mdx:19]: Well the other bullet points have the period outside of the hyperlink, so maybe just be consistent about it?

**Pull request merged**